frontend/containers/forms/sdgs/component.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```

Comment:
🟠

All the other form components are in `frontend/components/forms`. Shouldn't you move this one there too?

Reply:
I'm not sure.

I was under the impression that `components/` houses simple components that could be used in any project, while `containers/` houses project specific components. 

I do find it a little weird, but it would follow that pattern, if correct.

Reply:
Yes, you're right. `containers` should probably be used, nevertheless, we should have all the form components in one place. So either all in `components` or all in `containers`.

Reply:
Agreed. Given the options I think they should all live in `containers/`. 
We should wait until we aren't all of us actively working on forms as to prevent conflicts, then to the move. 

Merging for now as to not block the form being created.